crates/sui-framework/packages/move-stdlib/sources/vector.move

```diff
@@ -382,3 +382,91 @@ public macro fun zip_map_ref<$T1, $T2, $U>(
     zip_do_ref!($v1, $v2, |el1, el2| r.push_back($f(el1, el2)));
     r
 }
+
+/// Performs an in-place insertion sort on the vector `v` using the comparison function `le`.
+/// The sort is stable, meaning that equal elements will maintain their relative order.
+///
+/// Insertion sort is efficient for small vector, and can be faster than merge sort for almost
+/// sorted vectors (e.g. when the vector is already sorted or nearly sorted).
```

Comment:
Do we have have approximate numbers here to help folks understand what "small" means?

Reply:
32 was approximate number. Heavily depends on the set.